libcloud/common/base.py

```diff
@@ -867,6 +867,8 @@ def request(self, action, params=None, data=None, headers=None,
         return response
 
     def morph_action_hook(self, action):
+        if not action.startswith("/")

```

Comment:
btw, this is fixed now. although checks seem to be failing ;( not sure what that's about.

Let me know if there is anything else I can do.